test/base-warnings.js

```diff
@@ -0,0 +1,11 @@
+var tape = require('tape');
+var mjAPI = require("../lib/main.js");
+mjAPI.config({undefinedCharError: true});
+
+tape('basic test: check warnings', function (t) {
+    t.plan(2);
+    mjAPI.typeset({math:'呵', html:true})
+        .catch(errors => t.ok(errors, 'CommonHTML output reports error'));
+    mjAPI.typeset({math:'呵', svg:true})
```

Comment:
Because you can't specify the encoding used in a javascript source file, it is bad practice to use raw unicode (I assume this is UTF-8, but there are other unicode encodings).  For example, once of the editors I use will get this wrong, and were I to save it, the characters would change.

Were this to be loaded into a web page (and I know that it won't be), the javascript would be loaded (even in modern browsers, last time I checked) using the encoding of the web page itself, so if this were loaded into a non-UTF-8 page, it might not produce the correct character.  This is why we use explicit `\uXXXX` for the localization menu, for example.  It isn't about modern versus old browsers, it is about making assumptions about the encoding used.